projects/baby_driver/scripts/test_can_util.py

```diff
@@ -0,0 +1,53 @@
+""" This Module Tests methods in can_util.c """
+import unittest
+
+from can_util import can_pack
+
+class TestCanPack(unittest.TestCase):
+    """ Test Can Util data_pack function """
+
+    def test_low_bound(self):
+        """ Tests Maximum values """
+        self.assertEqual(b'\x00', can_pack(data_list=[(0, 1)]))
```

Comment:
Pro tip: you don't need to specify the argument name like `data_list=`, especially since it doesn't really add anything here.